dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -797,4 +797,24 @@ public Object put(String key, Object value) {
 	public boolean hasLiveVersion() throws DotStateException, DotDataException {
 		return APILocator.getVersionableAPI().hasLiveVersion(this);
 	}
+	
+	/**
+	 * Get the contentlet Content Type
+	 * @return the contentlet Content Type
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	public ContentType getContentType() throws DotDataException, DotSecurityException{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)